client/src/components/MediaCard/MediaCard.js

```diff
@@ -0,0 +1,31 @@
+
+import React from 'react';
+import { Link } from "react-router-dom";
+import "./MediaCard.css";
+
+
+// Function based React Component
+const MediaCard = (props) => {
+
+  // Default Class to apply to Component
+  let styles = {
+    height: '248px',
+    width: '248px',
+    backgroundImage: `url(${props.imageSrc})`,
+    backgroundSize: 'cover',
+    display: 'flex',
+    flexDirection: 'column-reverse'
+  }
```

Comment:
Include these styles in your MediaCard.css file, with the exception of your `backgroundImage`. You can still pass the background image in as props in the MediaCard component - it would looks something like this: `style={{backgroundImage: `url(${props.imageSrc})`}}`.